Makefile

```diff
@@ -1,9 +1,11 @@
 default: lintf
 
 PROTO_FILES=./protos/node/node.proto \
+			./protos/escrow/escrow.proto \
+			./protos/escrow/ledger.proto \
```

Comment:
golang would complain for putting the different packages in the same folder;
I'd suggest merging those into one if they are closely related, or split into two folders.

Reply:
moved to a separate folder, the ledger proto is copied from atlas-ledger, better not to merge to one because it can be change in that project.